### ISSUE TITLE
GraphQL support

### ISSUE THREAD

**Issue by insipx** [paritytech/substrate-archive#16]

**Comment by xlc** [comment]
We are using hasura for a similar setup we have and it works very well.

**Comment by 2075** [comment]
btw, i'm on it here: https://github.com/playzero/substrate-graph

**Comment by insipx** [comment]
Nice! GraphQL is a feature I'm excited about, so I'll definitely be watching the progress of substrate-graph.

Originally, I wanted to implement GraphQL support natively with Juniper: https://github.com/graphql-rust/juniper, and have it enabled with a compile-time feature if it's needed. Hasura seems a fine solution too, however.

Feel free to open issues and/or shoot me a message on riot if you run into any trouble

**Comment by 2075** [comment]
yes, hasura gained quite some traction and if you want maintainable bits and pieces it is imho an efficient way to go.

**Comment by 2075** [comment]
need to ask you about some bits, though. :D
